authors/apps/authentication/admin.py

```diff
@@ -0,0 +1,6 @@
+from django.contrib import admin
```

Comment:
Is this file related to the migration from 1.10 to 2.1??

Reply:
The reason why it was included was to give people access to the browsable version of the app so we can check whether the admin route works. It was not meant to be pushed so this was a slight error that will be fixed.